.github/workflows/release.yml

```diff
@@ -28,6 +28,15 @@ jobs:
         uses: actions/checkout@v2
       - name: Create release archive and notes
         run: .github/workflows/create_archive_and_notes.sh
+      - name: Publish wheel dist
+        env:
+          # This special value tells pypi that the user identity is supplied within the token
+          TWINE_USERNAME: __token__
+          # Note, the PYPI_API_TOKEN was added on
+          # https://github.com/bazelbuild/rules_python/settings/secrets/actions 
+          # and currently uses a token which authenticates as https://pypi.org/user/alexeagle/
```

Comment:
What's the long-term plan here to avoid using personal tokens?

Reply:
oh, Richard already fixed this to use a Google-owned token. I'll update the comment